acj/static/test/config/protractor_cucumber.js

```diff
@@ -17,8 +17,6 @@ exports.config = {
         'version': '52',
         'chromedriverVersion': '2.22',
         'selenium-version': '2.53.1',
-        'shardTestFiles': true,
```

Comment:
I accidentally included it but its something I wanted to change anyways. 

Running 3+ selenium instances gets the acceptance tests done slightly faster locally (takes off about a minute) but they are more error prone when your machine has a lot going on since it opens and closes a new browser instance for every feature file. Additionally, its been getting really annoying having the browser instances pop over my editor whenever they randomly decide to focus on themselves.